MariiaNovitska/HW2/natural number.py

```diff
@@ -0,0 +1,12 @@
+input_number = input("Write your number: ")
+
+a = list(input_number)
+product = int(a[0])*int(a[1])*int(a[2])*int(a[3])
+b = a[::-1]
+reverse_number = int(''.join(map(str, b))) 
```

Comment:
look at another way
print(f'Reversed number: {a[::-1]}')

Reply:
updeted this task